frontend/src/main/scala/bloop/engine/tasks/compilation/CompileResult.scala

```diff
@@ -138,8 +138,13 @@ object FinalCompileResult {
             case Compiler.Result.Empty => s"${projectName} (empty)"
             case Compiler.Result.Cancelled(problems, ms, _) =>
               s"${projectName} (cancelled, failed with ${Problem.count(problems)}, ${ms}ms)"
-            case Compiler.Result.Success(_, _, _, ms, _, isNoOp) =>
-              val noOp = if (isNoOp) " no-op" else ""
+            case Compiler.Result.Success(_, _, _, ms, _, isNoOp, reportedFatalWarnings) =>
+              val noOp = {
```

Comment:
`noOp` no longer seems a valid name

```suggestion
              val mode = {
```

Reply:
Will change manually as it requires updating its reference.